### PR TITLE
Add chat title bar, Add service message

### PR DESCRIPTION
**Pull request by Muaath5**
**What new features or bug fixes does this pull request introduce? Explain.**
Chat title bar

**Is this pull request related to an existing issue?**
No

**Does this pull request comply with the conditions described in contribution [guidelines](https://github.com/DIBgram/DIBgram/blob/main/CONTRIBUTING.md#submitting-a-pull-request)?**

- [x] There are no errors, warnings or notices from ESLint or StyleLint
- [x] It compiles and runs successfully
- [x] It does not introduce a feature that is not present in the official Telegram Desktop app
- [x] Behavior and UI is similar to the official Telegram Desktop app

### CODE REVIEW

**Comment by Mammad900** [comment]
@Muaath5 What happened on src/dibgram/auth/auth-step/cloud-password/cloud-password.js?

**Comment by Muaath5** [comment]
@Mammad900 Nothing, I don't know why GitHub marked it as different.

**Comment by Muaath5** [comment]
@Mammad900 I think last line (\n)

**Comment by Muaath5** [comment]
Now it should be as you wanted.
Only needs some working on ThreeDotsButton & other icon buttons.

And a bit of working on subscribers/members label, Because it's currently shows 5 all the time.

And ProfilePhoto needed

**Comment by Mammad900** [comment]
Hi @Muaath5. Unfortunately, the internals of this part of the app is too different from your pull request (this is **not** your fault)

So, I'll implement these parts myself and close this pull request.  
I may use your contributed code in my commits. If I do, I'll add you as a co-author.

File: src/dibgram/messenger/sections/chat/headers/title.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import './title.scss';
-
+import { __pl } from '../../../../language-pack/language-pack';
 export default function TitleHeader({chat}) {
```
Comment (Mammad900) [inline src/dibgram/messenger/sections/chat/headers/title.js:5]: Please add an empty line after the imports. This will make the code easier to read.

**Comment by Mammad900** [comment]
Can you create another pull request? My conflicting commit seems to have messed up this one.

You can fork again, apply your changes and pull request.

**Pull request closed without merging**